configs/peer/genesis.json

```diff
@@ -89,6 +89,21 @@
           }
         }
       },
+      {
+        "Grant": {
```

Comment:
this is ok for now but make note that we should handle permissions with more care